tasks/helpers/responsehelper.js

```diff
@@ -0,0 +1,16 @@
+const responseHelper = `
+    /**
+     * @static
+     * @param {response}  response object
+     * @param {message}  message
+     * @param {data}  data
+     * @param {status}  status code
+     * @returns {object} object
+    */
+    export default (res, message, data, status, data) => {
```

Comment:
Then, we can do away with one `data` argument here. 
I think it is the one after `message` argument.